packages/mobile/src/send/utils.ts

```diff
@@ -188,34 +184,32 @@ export function* handleSendPaymentData(
   }
   yield put(storeLatestInRecents(recipient))
 
-  if (data.currencyCode) {
```

Comment:
So this was the root of the problem, right?

Reply:
Mhm, I think `selectPreferredCurrency` was being called just as a temp workaround for the grameen pilot